src/coreclr/nativeaot/System.Private.Reflection.Execution/src/Internal/Reflection/Execution/ExecutionEnvironmentImplementation.ManifestResources.cs

```diff
@@ -97,7 +97,7 @@ private static LowLevelDictionary<string, LowLevelList<ResourceInfo>> ExtractedR
                     // Lazily create the extracted resource dictionary. If two threads race here, we may construct two dictionaries
```

Comment:
I think the question to ask here is whether we need to create this global cache in the first place.